process.c

```diff
@@ -177,6 +177,9 @@ static void check_uid_switch(void);
 static void check_gid_switch(void);
 static int exec_async_signal_safe(const struct rb_execarg *, char *, size_t);
 
+VALUE rb_envtbl(void);
+VALUE rb_env_to_hash(void);
+
```

Comment:
Should this go in `internal/hash.h`?

Reply:
I don't have a problem with that if there are multiple `.c` files that need it.  I think currently there is only one, and therefore adding the prototype only to the `.c` file that is needed should be fine.  Certainly there are other places in CRuby where we include prototypes directly in `.c` files.